### ISSUE TITLE
2.0.6/7 Beta 2/3

### ISSUE THREAD

**Issue by Xetaxheb** [TanninOne/modorganizer#977]
Tldr: skse loads, throws error, profile ini's don't load but defaults (my docs) do, skse and mods work(s) fine anyway (as far as I can tell with minimal testing)

Running a basic mod list now, but issue existed on fresh vanilla install. (i've played and modded skyrim for thousands of hours)

`Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	skse_loader.exe
  Application Version:	0.1.7.3
  Application Timestamp:	55611f7d
  Fault Module Name:	usvfs_x86.dll
  Fault Module Version:	0.2.2.0
  Fault Module Timestamp:	57480628
  Exception Code:	c0000005
  Exception Offset:	0006dfd5
  OS Version:	6.1.7601.2.1.0.256.48
  Locale ID:	1033
  Additional Information 1:	ca43
  Additional Information 2:	ca43cbabe0fdd196a8cac0eb5bb68eec
  Additional Information 3:	894e
  Additional Information 4:	894e304147829a1fdbac2084d7b6fd7a
`

Latest skse(skse_1_07_03) selected, click play, skse cmd pops up, almost instant hang with that error.... except clicking "close this program" (in which case instantly thereafter:) or after a few seconds, skyrim runs normally. Mods and mcm's and such seem to work fine (as does getskseversion); but profile-directory ini's are not loading as they should be. The ones in my documents are. (easy test, have one set for fullscreen and the other not.)

**Probably unrelated** but perhaps caused by nvidia drivers and more specifically gameready optimization via geforce experience (new geforce 3.x is cancerous)? (it reads default directory ini settings when the window has focus with the game selected, and changes them when you click optimize) I can't test this further than closing geforce-experience's window and exiting the tray, as nvidia is extremely persistent about its processes running (close one and it comes back instantly).
